internal/bridgeapi/login.go

```diff
@@ -190,7 +230,23 @@ func SetLogin(cp CredentialProvider, authBaseURL *url.URL) {
 	}
 }
 
+// Resets the LoginManager state to nearly new, ready for a new SetLogin call
+func UnsetLogin() {
```

Comment:
`UnsetLogin` and `Logout` - which function is more suitable for the DBaaS controllers for invalidating tokens.

Reply:
_it _depends__

Both invalidate the current login state by forgetting the current token. However, `UnsetLogin` resets the CredentialProvider, which means `SetLogin` *must* be called with valid credentials before anything will work again.
`Logout` leaves the credential provider in place, which will be used on the next call to any of the client data functions. That is, any attempt to contact the API at that point will login with the credentials from the provider, which would be the kube secret specified for the credential provider.